### DISCUSSION TITLE
Is there a way to connect `wagmi/chains` to `tron.network`?

### DISCUSSION THREAD

**Discussion by hhun** [wevm/wagmi#3132] [Q&A]
`wagmi/chains` 有办法连接 `tron.network` 吗？
Is there a way to connect `wagmi/chains` to `tron.network`?

**Comment by eldiyar-dev** [comment]
Someone can give answer ?

**Comment by Canazo** [comment]
I'm trying to integrate wagmi with Tron Network.  Any light would be highly appreciated!

**Reply by RedWilly** [reply]
Do you have any update on this?

**Comment by dylanngph** [comment]
Pump this up
